extensions/hadoop-dist/hadoop-all/pom.xml

```diff
@@ -37,6 +37,18 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
+
+                <configuration>
+                    <shadeTestJar>false</shadeTestJar>
+                    <filters combine.children="append">
+                        <filter>
+                            <artifact>*:*</artifact>
+                            <excludes>
+                                <exclude>META-INF/MANIFEST.MF</exclude>
```

Comment:
Have you considered using the [ManifestResourceTransformer](https://maven.apache.org/plugins/maven-shade-plugin/examples/resource-transformers.html#ManifestResourceTransformer) instead?

Reply:
TBH I simply reused the same method we use for other dependencies explicitly